src/olympia/files/decorators.py

```diff
@@ -54,7 +54,7 @@ def last_modified(request, obj, key=None, **kw):
 
 
 def etag(request, obj, key=None, **kw):
-    return _get_value(obj, key, 'md5')
+    return _get_value(obj, key, 'sha256')
```

Comment:
Funny enough gce still uses crc32 in some cases for etags and aws also fully uses md5 hashes. Anyhow... the spec doesn't say what should be used so we're fine here.